ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainService.kt

```diff
@@ -238,6 +238,15 @@ class MainService : Service() {
     }
 
     override fun onDestroy() {
+        serviceMap.forEach { service ->
+            val pipeline = service.value.pipeline
+            pipeline.stop()
```

Comment:
https://github.com/nnstreamer/nnstreamer-android/blob/0b565d02ebe4f697dc1dde659c4a13ddd2b45057/ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainService.kt#L89

There is no problem if `close` is called immediately, but the reason for stopping and then closing is to unregister the service from nsdManager through the callback function.